sdk/servicebus/service-bus/karma.conf.js

```diff
@@ -49,21 +49,21 @@ module.exports = function(config) {
     // https://www.npmjs.com/package/karma-env-preprocessor
     envPreprocessor: [
       "SERVICEBUS_CONNECTION_STRING",
-      "QUEUE_NAME",
-      "QUEUE_NAME_NO_PARTITION",
-      "QUEUE_NAME_SESSION",
-      "QUEUE_NAME_NO_PARTITION_SESSION",
-      "TOPIC_NAME",
-      "TOPIC_NAME_NO_PARTITION",
-      "TOPIC_NAME_SESSION",
-      "TOPIC_NAME_NO_PARTITION_SESSION",
-      "SUBSCRIPTION_NAME",
-      "SUBSCRIPTION_NAME_NO_PARTITION",
-      "SUBSCRIPTION_NAME_SESSION",
-      "SUBSCRIPTION_NAME_NO_PARTITION_SESSION",
-      "TOPIC_FILTER_NAME",
-      "TOPIC_FILTER_SUBSCRIPTION_NAME",
-      "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
+      "QUEUE_NAME_BROWSER",
+      "QUEUE_NAME_NO_PARTITION_BROWSER",
+      "QUEUE_NAME_SESSION_BROWSER",
+      "QUEUE_NAME_NO_PARTITION_SESSION_BROWSER",
+      "TOPIC_NAME_BROWSER",
+      "TOPIC_NAME_NO_PARTITION_BROWSER",
+      "TOPIC_NAME_SESSION_BROWSER",
+      "TOPIC_NAME_NO_PARTITION_SESSION_BROWSER",
+      "SUBSCRIPTION_NAME_BROWSER",
+      "SUBSCRIPTION_NAME_NO_PARTITION_BROWSER",
+      "SUBSCRIPTION_NAME_SESSION_BROWSER",
+      "SUBSCRIPTION_NAME_NO_PARTITION_SESSION_BROWSER",
```

Comment:
Why do we have 4 different subscriptions? And why do they need to be different for node vs browser?

Reply:
Some tests might use them.
Goal was to have separate copy of node entities to use in browser.
Do you mean yo go through all test-cases and optimize the entities? Not sure if that's a good use of time.

Reply:
lol, I misunderstood the "Service Bus **Subscription**" entity with "Azure **Subscription**"
The latter should be re-used across all variations of tests and platforms.
The former needs the variations of partition/un-partition sessions/no-sesssions and node/browser